websockets/client.py

```diff
@@ -30,9 +31,17 @@ class WebSocketClientProtocol(WebSocketCommonProtocol):
 
     def __init__(self, *,
                  origin=None, extensions=None, subprotocols=None,
-                 extra_headers=None, **kwds):
+                 extra_headers=None, use_compression=True, **kwds):
         self.origin = origin
-        self.available_extensions = extensions
+        self.available_extensions = []
+        if use_compression:
+            self.available_extensions.append(
+                'permessage-deflate; '
+                'client_no_context_takeover; '
```

Comment:
The code I added support both modes, with and without retaining the sliding window. I put that as a default client side config as it is what I get from chrome client or wscat, nothing more. In any case, this will disappear once the code is generalized.